app/models/item.rb

```diff
@@ -3,4 +3,6 @@ class Item < ApplicationRecord
   belongs_to :merchant
   has_many :invoice_items
   has_many :invoices, through: :invoice_items
+
+  enum status: { 'enabled': 0, 'disabled': 1 }
```

Comment:
Just remembered, we will have to write a test for the new column since it's an enum.